react/week4/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./App.css";
+import StateProvider from "./context.js";
```

Comment:
See if you can come up with more descriptive names. State and Main are very generic names

Reply:
Are the recent names better now?